tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -211,12 +211,23 @@ function configure_containerd() {
 
 function configure_kata() {
 	if [ "${CONFIGURE_CC:-}" == "yes" ]; then
+		cc_config="/opt/kata/share/defaults/kata-containers/configuration-cc.toml"
 		sed -E \
-			-e 's#^image = .+#initrd = "/opt/kata/share/kata-containers/kata-containers-initrd.img"#' \
-			-e 's#^(kernel_params = .+)"#\1 agent.config_file=/etc/kata-config.toml"#' \
+			-e 's#^(kernel_params = .+)"#\1 agent.config_file=/etc/kata-containers/agent.toml"#' \
 			-e 's#.*service_offload = .+#service_offload = true#' \
 			"/opt/kata/share/defaults/kata-containers/configuration-qemu.toml" > \
-			"/opt/kata/share/defaults/kata-containers/configuration-cc.toml"
+			"$cc_config"
+		if [ "$(uname -m)" == "s390x" ]; then
+			sed -Ei \
+				-e 's#^kernel = .+#kernel = "/opt/kata/share/kata-containers/kata-containers-secure.img"#' \
+				-e '/^image = .+/d' \
+				-e 's/^(# )?confidential_guest = .+/confidential_guest = true/' \
+				-e 's#^shared_fs = .+#shared_fs = "virtio-9p"#' \
+				"$cc_config"
+				# TODO remove 9p workaround once https://bugzilla.linux.ibm.com/show_bug.cgi?id=194746 is resolved
+		else
+			sed -Ei 's#^image = .+#initrd = "/opt/kata/share/kata-containers/kata-containers-initrd.img"#' "$cc_config"
```

Comment:
It might be good to rework slightly so that this path be specified as `"$protimg_kata_dir/kata-containers-initrd.img"`.

Reply:
Same comment for the previous `sed` call.